src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -63,7 +64,12 @@ async function initializeAudio() {
     },
   });
 
-  const micSourceNode = context.createMediaStreamSource(micStream);
+  const micSourceNode = new MediaStreamAudioSourceNode(context, 
+      { mediaStream: micStream });
```

Comment:
https://google.github.io/styleguide/jsguide.html#formatting-object-literals

```
{mediaStream:micStream});
```

(with no space between braces and its content.